static/app/components/events/interfaces/frame/deprecatedLine.tsx

```diff
@@ -270,10 +278,39 @@ export class DeprecatedLine extends Component<Props, State> {
     return null;
   }
 
+  renderShowHideToggle(onShowFramesToggle) {
+    const hiddenFrameCount = this.props.hiddenFrameCount;
+    if (hiddenFrameCount) {
+      return (
+        <a
+          onClick={e => {
+            onShowFramesToggle(e);
+          }}
+        >
+          {this.props.isToggled ? 'Hide' : 'Show'} {hiddenFrameCount} more{' '}
```

Comment:
We want this to be translated to other languages, so we should use `tn` from `sentry/locale`. You can see an equivalent usage here: https://github.com/getsentry/sentry/blob/09f3e322b73c4aae07c235973205e4558549b23e/static/app/components/events/interfaces/crashContent/exception/content.tsx#L114-L116